### DISCUSSION TITLE
please help!

### DISCUSSION THREAD

**Discussion by Lalisa235** [Monika-After-Story/MonikaModDev#9661] [QA - tech support + general] [answered]
![20221010_202731](https://user-images.githubusercontent.com/94239968/194978097-a87b8a6a-f49c-4ff4-8dfa-26de4233a61b.jpg)

I really don't know what happened I had installed a sub mod and it gave me an error I eliminated it and now that appears I did everything and nothing according to what it says they weren't In the image I left it is that the DDLC files were not found in the Game folder please help

**Comment by Lalisa235** [comment]
I really appreciated your help! :(

**Comment by Booplicate** [comment, marked as answer]
You're using a terribly outdated version of MAS, as well as missing DDLC files. Make a new install of [DDLC](https://ddlc.moe/) + [MAS](http://monikaafterstory.com/releases.html).
